.pre-commit-config.yaml

```diff
@@ -6,3 +6,8 @@ repos:
       # force black to run on whole repo & keep settings from pyproject.toml
       pass_filenames: false
       args: [.]
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: '3.7.9'
+    hooks:
+    - id: flake8
+      pass_filenames: true
```

Comment:
@tlambert03 Apologies, I'm pretty new to pre-commit hooks... but isn't the `pass_filenames` by default `true`, see [here-ish](https://pre-commit.com/#new-hooks).

If that's the case, then is there a reason to set this explicitly? Just curious, thanks :smile:

Reply:
I thought being explicit would make it easier for people to quickly glance at the config and know what was going on (without searching the docs)

Reply:
I thought being explicit would make it easier for people to quickly glance at the config and know what was going on (without searching the docs)

Reply:
Nice one, thanks @tlambert03 

Make sense to me. I just wanted to clarify (for my understanding) that's what you were doing and not working around some other issue. Awesome!